### PR TITLE
matterbridge: Add version 1.22.0

### PR DESCRIPTION
**Pull request by 42wim**

### CODE REVIEW

File: bucket/matterbridge.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "version": "1.22.0",
+    "description": "Bridge between mattermost, IRC, gitter, xmpp, slack, discord, telegram, rocketchat, twitch, ssh-chat, zulip, whatsapp, keybase, matrix, microsoft teams, nextcloud, mumble, vk and more with REST API.",
```
Comment (Ash258) [inline bucket/matterbridge.json:3]: - Nonsense to list random list of X chat applications
- Do not explain what this application do

File: bucket/matterbridge.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            "url": "https://github.com/42wim/matterbridge/releases/download/v1.22.0/matterbridge-1.22.0-windows-32bit.exe#/matterbridge.exe",
+            "hash": "b855642ba0bf081c7356745fe8bebd69147f6ec7effb6b9210416bd930ba5a2d"
+        }
+    },
+    "bin": "matterbridge.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-64bit.exe"
```
Comment (Ash258) [inline bucket/matterbridge.json:21]:
```suggestion
                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-64bit.exe#/matterbridge.exe"
```

File: bucket/matterbridge.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            },
+            "32bit": {
+                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-32bit.exe"
```
Comment (Ash258) [inline bucket/matterbridge.json:24]:
```suggestion
                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-32bit.exe#/matterbridge.exe"
```

File: bucket/matterbridge.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            }
+        }
```
Comment (Ash258) [inline bucket/matterbridge.json:26]: Add hash extraction

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/matterbridge.json (hunk 1 above)
Comment (42wim) [reply bucket/matterbridge.json:3]: Not sure what you want me to write there, the application bridges or creates a bridge betwen those list of X chat applications.

File: bucket/matterbridge.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-64bit.exe#/matterbridge.exe"
+            },
+            "32bit": {
+                "url": "https://github.com/42wim/matterbridge/releases/download/v$version/matterbridge-$version-windows-32bit.exe#/matterbridge.exe"
+            }
+        }
```
Comment (Ash258) [inline bucket/matterbridge.json:26]:
```suggestion
        },
        "hash": {
            "url": "$baseurl/checksums.txt"
        }
```

File: bucket/matterbridge.json (hunk 1 above)
Comment (Ash258) [inline bucket/matterbridge.json:3]:
```suggestion
    "description": "Bridges between a growing number of protocols",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**